Reason: Skip test that works only with amd64 platform
Upstream: No

diff --git a/pkg/util/utils_test.go b/pkg/util/utils_test.go
index 3e5e36965..1d242ce37 100644
--- a/pkg/util/utils_test.go
+++ b/pkg/util/utils_test.go
@@ -28,6 +28,7 @@ import (
 )
 
 func TestGetBinaryDownloadURL(t *testing.T) {
+	t.Skip("amd64-only test")
 	testData := []struct {
 		version     string
 		platform    string
